assets/translations/en-us.yaml

```diff
@@ -3299,7 +3335,7 @@ wizard:
   back: Back
```

Comment:
Not new apparently but `Next` and `Back` don't go together.  Next/Previous or Forward/Back.

Reply:
Fixed